bookserver/crud.py

```diff
@@ -115,22 +115,21 @@ async def fetch_last_answer_table_entry(
 
 # Courses
 # -------
-async def fetch_course(course_name: str) -> CoursesValidator:
-    query = select(Courses).where(Courses.course_name == course_name)
+async def fetch_base_course(base_course: str) -> CoursesValidator:
```

Comment:
fetching a course is a necessary function.  not all courses are base courses.

Reply:
We should probably have another function -- `fetch_course` vs `fetch_base_course`. So far, we're only using `fetch_base_course`, so I just renamed it. I'd suggest postponing `fetch_course` until we have a need for it.

Reply:
Well, I knew the need was coming soon, so thats why I added the fetch_course when I wrote create.  Its early on with lots of things coming fast, so everything is not always going to end up perfectly in sync.